docs/content/3_blog/2_twill-3-introducing-oop-builders.md

```diff
@@ -212,7 +212,7 @@ similar to that of the settings registration. In the app service provider we can
 TwillNavigation::addLink(
     NavigationLink::make()->forModule('blogs')->title('Blogs')
         ->setChildren([
-            NavigationLink::make()->forRoute('twill.app.settings.page', ['group' => 'blog'])->title('Blog settings'),
+            NavigationLink::make()->forRoute(config('twill.admin_route_name_prefix') . 'app.settings.page', ['group' => 'blog'])->title('Blog settings'),
```

Comment:
I'm not sure this one should change since it is a snippet of code that's supposed to be used in a Twill app with the standard prefix.

Reply:
Yeah I was hesitant about that one personally, but figured it would be best practices to have in the documentation for any maintainers to consider when referencing the docs. Extensibility is always a good consideration.